irma-core/index.js

```diff
@@ -1,12 +1,10 @@
 const StateMachine = require('./state-machine');
-const userAgent    = require('./user-agent');
```

Comment:
Is it logical to move this to `irma-web`?

Reply:
No it is not, there I apparently moved it to `irma-client` and I remembered it wrong . In `irma-client` makes more sense.